# HG changeset patch
# User Rik <rik@octave.org>
# Date 1385829744 28800
#      Sat Nov 30 08:42:24 2013 -0800
# Branch stable
# Node ID 8bc09600f0ed0ad2cb31c99a5da38d1125190532
# Parent  482cc05b4f99e469d4d1ccc05a39b7602cc16b73
Correct spelling in cset 482cc05b4f99.

* NEWS: Correct spelling of belive.

* main-window.c: Correct spelling of belive.c

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -8,17 +8,17 @@ Summary of important user-visible change
      decided to wait until the 4.0.x release series before making the
      GUI the default interface (until then, you can use the --force-gui
      option to start the GUI).
 
      Given the length of time and the number of bug fixes and
      improvements since the last major release Octave, we also decided
      against delaying the release of all these new improvements any
      longer just to perfect the GUI.  So please enjoy the 3.8 release of
-     Octave and the preview of the new GUI.  We beleive it is working
+     Octave and the preview of the new GUI.  We believe it is working
      reasonably well, but we also know that there are some obvious rough
      spots and many things that could be improved.
 
      WE NEED YOUR HELP.  There are many ways that you can help us fix
      the remaining problems, complete the GUI, and improve the overall
      user experience for both novices and experts alike:
 
        * If you are a skilled software developer, you can help by
diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -1897,17 +1897,17 @@ main_window::show_gui_info (void)
          "But because it is not quite as polished as we would like, we "
          "have decided to wait until the 4.0.x release series before "
          "making the GUI the default interface.</p>"
          "<p>Given the length of time and the number of bug fixes and "
          "improvements since the last major release Octave, we also "
          "decided against delaying the release of all these new "
          "improvements any longer just to perfect the GUI.  So please "
          "enjoy the 3.8 release of Octave and the preview of the new GUI.  "
-         "We beleive it is working reasonably well, but we also know that "
+         "We believe it is working reasonably well, but we also know that "
          "there are some obvious rough spots and many things that could be "
          "improved.</p>"
          "<p><strong>We Need Your Help</strong></p>"
          "<p>There are many ways that you can help us fix the remaining "
          "problems, complete the GUI, and improve the overall user "
          "experience for both novices and experts alike (links will open "
          "an external browser):</p>"
          "<p><ul><li>If you are a skilled software developer, you can "
